### PR TITLE
pytest/test-submit

### PR DESCRIPTION
**Pull request by alphasentaurii**
- Converted test_submit.py to pytest
- added 'submit' test marker

### CODE REVIEW

**Review by stscieisenhamer — APPROVED** [review]

**Comment by zacharyburnett** [comment]
it looks like there's a network issue connecting to the resource: 
```
=========================== short test summary info ============================
ERROR test/submit/test_submit.py - urllib.error.URLError: <urlopen error [Errno -5] No address associated with hostname>
!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!
========================= 2 warnings, 1 error in 5.40s =========================
```

**Comment by alphasentaurii** [comment]
> it looks like there's a network issue connecting to the resource:
> 
> ```
> =========================== short test summary info ============================
> ERROR test/submit/test_submit.py - urllib.error.URLError: <urlopen error [Errno -5] No address associated with hostname>
> !!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!
> ========================= 2 warnings, 1 error in 5.40s =========================
> ```

The original tests patched the urlopen functionality via mock to a file (yaml-based form) so it's not supposed to need an actual network connection. However as far as I can tell none of the tests actually make use of this so I'm not sure it even needs to be included. For now I've commented it out to see if that resolves the issue. We may want to just remove it altogether.

File: test/submit/test_submit.py [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+import os
+from crds.submit import Submission, NoFilesSelected
+
+
+@mark.submit
+class TestSubmit:
+
+    # Instantiate the Submission object used in these tests:
+    s = Submission('hst', 'dev', context='hst_0723.pmap')
+
```
Comment (hbrown-ST) [inline test/submit/test_submit.py:11]: Still need these?

File: test/submit/test_submit.py [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+    # @fixture()
```
Comment (hbrown-ST) [inline test/submit/test_submit.py:12]: Still need these commented lines?

**Review by zacharyburnett — APPROVED** [review]
looks like the tests pass now!

**Pull request merged**